src/handlers/beta_backport.rs

```diff
@@ -0,0 +1,85 @@
+use crate::config::BetaBackportConfig;
+use crate::github::{IssuesAction, IssuesEvent, Label};
+use crate::handlers::Context;
+use regex::Regex;
+
+lazy_static! {
+    // See https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-issues/linking-a-pull-request-to-an-issue
+    static ref CLOSES_ISSUE: Regex = Regex::new("(close[sd]|fix(e[sd])?|resolve[sd]) #(\\d+)").unwrap();
+}
+
```

Comment:
Is this going to trigger a beta-backport nomination for *any* PR that is fixing an issue, no matter if the originating issue is a regression and the channel (nightly, beta or stable) where the regression happened?

(sorry if I missing some context)

Reply:
cc @mibac138